test/suites/remote.sh

```diff
@@ -14,6 +14,10 @@ test_remote_url() {
 
   # an invalid protocol returns an error
   ! lxc_remote remote add test "${url}" --accept-certificate --password foo --protocol foo || false
+  # a connectivity issue returns an error
```

Comment:
Please move these down to the section starting at line 28, this way its wrapped in the LXD_OFFLINE check and doesn't interrupt the flow of the testing of the urls list here.